projects/1.20-fabric/assets/mythic-mobs/mythicmobs/lang/zh_cn.json

```
@@ -0,0 +1,68 @@
+{
+  "advancements.mythicmobs.title": "神话生物",
+  "advancements.mythicmobs.description": "一个神话世界!",
```

```suggestion
  "advancements.mythicmobs.description": "一个神话世界！",
```
除patchouli格式符号外，应使用全角符号，请全文通查